resources/resources.go

```diff
@@ -652,6 +652,65 @@ type Messageinformation struct {
 	SpamAssassinScore float64            `mailjet:"read_only"`
 }
 
+// Email struct handle users input
+type Email struct {
+	Email string
+	Name  string
+}
+
+// Emails is a collection of emails
+type Emails []Email
+
+// Attachment struct represent a content attachment
+type Attachment struct {
+	ContentType   string `json:"ContentType"`
+	Base64Content string `json:"Base64Content"`
+	Filename      string `json:"Filename,omitempty"`
+}
+
+// Attachments collection
+type Attachments []Attachment
+
+// InlinedAttachments collection
+type InlinedAttachments []InlineAttachment
+
+// InlineAttachment struct represent the content of an inline attachement
+type InlineAttachment struct {
+	Attachment
+	ContentID string `json:"ContentID,omitempty"`
+}
+
+// MessagesInput represents the payload input taken by send API v3.1
+type MessagesInput struct {
+	From                     Email
+	To                       Emails
```

Comment:
Since they are always mendatory in the specs I haven't mark them as omitempty so when the struct will be serialised, the payload will always contain these fields